File: Programming/Practice/12/Python/12.py
number = input("Введите число, от которого нужно нужно взять факториал: ")
if int(number) > 0:
	i = 1
	n = 1
	while i < int(number):
		n *= (i + 1)
		i += 1
	if n > 1000000000:
		print("n Не соответствует условию (1 <= n <= 1000000000)")
	else:
		print(n)
else:
	print("Неправильный ввод")

File: Programming/Practice/12/C++/12/12/12.cpp
#include<iostream>

using namespace std;

int main()
{
	int number;
	setlocale(LC_ALL, "RUS");
	cout << "Введите число, от которого нужно нужно взять факториал: " << endl;
	cin >> number;
	int n = 1;
	if (number > 0)
	{
		for (int i = 1; i < number; i++)
		{
			n = n * (i + 1);
		}
		if ((n >= 1) && (n <= 1000000000))
		{
			cout << n;
		}
		else
		{
			cout << "Факториал не соответствует условию (1 <= n <= 1000000000)" << endl;
		}
	}
	else
	{
		cout << "Неправильный ввод" << endl;
	}
}